create lw13 hackathon event (#30912)
supabase/supabase: apps/www/_events/2024-12-05-launch-week-13-hackathon.mdx

---
title: 'Supabase Launch Week 13 Hackathon'
meta_title: 'Supabase Launch Week 13 Hackathon'
subtitle: 'Build an Open Source Project over 10 days. 5 prize categories.'
meta_description: 'Build an Open Source Project over 10 days. 5 prize categories.'
type: 'hackathon'
onDemand: false

> date: '2024-12-05T09:00:00.000-07:00'

Reviewer: dec 6?

```suggestion
date: '2024-12-06T09:00:00.000-07:00'
```

timezone: 'America/Los_Angeles'
duration: '10 days'
categories:
  - hackathon
main_cta:
  {
    url: 'https://hackathon.dev/lw13',
    target: '_blank',
    label: 'Submit project'
  }
---

> The hackathon starts on Friday, December 5th at 09:00 am PT and ends on Sunday, December 15th at 11:59 pm PT. You could win an extremely limited edition Supabase swag and add your name to the Supabase Hackathon Hall of Fame.

Reviewer: ```suggestion
The hackathon starts on Friday, December 6th at 09:00 am PT and ends on Sunday, December 15th at 11:59 pm PT. You could win an extremely limited edition Supabase swag and add your name to the Supabase Hackathon Hall of Fame.
```

For some inspiration, check out all the [winners from previous hackathons](https://supabase.com/blog/tags/hackathon).

This is the perfect excuse to "Build in a weekend, scale to millions.” Since you retain all the rights to your submissions, you can use the hackathon as a launch pad for your new Startup ideas, side projects, or indie hacks.

## Key Facts

- You have 10 days to build a new **Open Source** project using Supabase in some capacity

>   - Starting 09:00 am PT Friday, December 5th 2024

Reviewer: ```suggestion
  - Starting 09:00 am PT Friday, December 6th 2024
```

  - The submission deadline is 11:59 pm Sunday midnight PT, December 15th 2024
- Enter as an individual or as a team of up to 4 people
- Build whatever you want - a project, app, tool, library. Anything.
- 1-minute video containing the following:
  - Name of the project
  - Demonstration of the project
  - How Supabase is used within the project
- [Here is an example video](https://youtu.be/KaWJQzTTx5k). We do not assess the quality of the video itself. Remember to keep it concise.

## Prizes

There are 5 categories there will be prizes for:

> - Best overall project (Swag kit)
> - Best use of AI (Swag kit)
> - Most fun / best easter egg (Swag kit)
> - Most technically impressive (Swag kit)
> - Most visually pleasing (Swag kit)

Reviewer 2: ```suggestion
- Best overall project (Swag kit)
- Best use of AI (Swag kit)
- Most fun / best easter egg (Swag kit)
- Most visually pleasing (Swag kit)
- Best project built with bolt.new (Swag kit)
```

Reviewer: taken care of here: https://github.com/supabase/supabase/pull/30973

There will be a winner and a runner-up prize for each category. Every team member on winning/runner-up teams gets a Supabase Launch Week swag kit.

## Submission

You should submit your project using [this form](https://hackathon.dev/lw13) before 11:59 pm Sunday midnight PT, December 15th, 2024.

## Judges

The Supabase team will judge the winners for each category.
We will be looking for:

- creativity/inventiveness
- functions correctly/smoothly
- visually pleasing
- technically impressive
- use of Supabase features
- FUN! 😃

## Rules

- Team size 1-4 (all team members on winning teams will receive a prize)
- You cannot be on multiple teams
- One submission per team
- It's not a requirement to use AI
- All design elements, code, etc., for your project must be created **during** the event
- All entries must be Open Source (link to source code required in entry)
- Must use Supabase in some capacity

> - Can be any language or framework

Reviewer: ```suggestion
- Can use any language or framework
```
